screens/AnnouncementScreen.js

```diff
@@ -21,9 +21,32 @@ class AnnouncementScreen extends React.Component {
   }
 
   render() {
+    // const {goBack} = this.props.navigation;
     return (
       <View>
-        <Hamburger navigation={this.props.navigation} />
+        <TouchableOpacity
```

Comment:
Think there's potential for this button to be made into a (styled-)component given how verbose it looks in the current code + possibility of re-use in other screens if DC chooses to add additional screens in their sidebar.

Reply:
Added as a todo for now